tests/e2e/lib/dpa_helpers.go

```diff
@@ -72,7 +73,7 @@ func (v *DpaCustomResource) Build(backupRestoreType BackupRestoreType) error {
 					DefaultPlugins: v.CustomResource.Spec.Configuration.Velero.DefaultPlugins,
 				},
 				NodeAgent: &oadpv1alpha1.NodeAgentConfig{
-					UploaderType: "restic",
+					UploaderType: string(backupRestoreType),
```

Comment:
This will not break? the accepted values are restic and kopia but the type can be csi, csi-datamover, restic or kopia